test/test_sparse_csr.py

```diff
@@ -2173,25 +2165,18 @@ def test_dense_to_from_sparse_compressed(self, device, layout):
         if layout is torch.sparse_bsc:
             # TODO: Remove this once support has been enabled
             return
-        if layout is torch.sparse_bsr:
-            # TODO: Remove this once support has been enabled
-            return
 
-        for shape in [(0, 10), (6, 0), (6, 10), (0, 0)]:
+        blocksizes = [(2, 2)]
+        if layout is torch.sparse_bsr:
+            blocksizes += [(3, 5), (6, 10)]
+        for shape, blocksize in itertools.product([(0, 10), (6, 0), (6, 10), (0, 0)], blocksizes):
             dense = make_tensor(shape, dtype=torch.float, device=device)
             dense = dense.relu()  # Introduce some sparsity
-            sp_matrix = self._construct_sp_matrix(dense, layout)
-            pt_matrix = self._convert_to_layout(dense, layout)
+            print("shape: ", shape, " blocksize: ", blocksize, " dense: ", dense)
```

Comment:
I left this for better debugging a spurious CI failure. Will remove before merging.